developing-packages.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: page
+title: Developing Packages & Plugins
+permalink: /developing-packages/
+---
+
+* TOC
+{:toc}
+
+## Package introduction
+
+Dart packages enable the creation of modular code that can easily be shared. A
```

Comment:
Yes, I will keep it. There are also java, swift, etc. packages that a plugin developer will be familiar with.